lib/init/grass.html

```diff
@@ -21,19 +21,19 @@ <h3>Flags:</h3>
 <dd> Prints the version of GRASS and exits
 
 <dt><b>-c</b>
-<dd> Creates new GRASS unprojected location in specified GISDBASE
+<dd> Creates new GRASS project (location) without coordinate reference system in specified GISDBASE
```

Comment:
I have couple of issues with this, none of them directly related to the main goals of the PR.

1. `-c` for project/location creation gives `ERROR: Provide CRS to create a location` in 8.2. `-c XY` is what is needed for the described behavior. This might be there since 7.0.
2. I think XY is a CRS, no? It is just not georeferenced. Any opinions on this? Definitions?
3. The original "unprojected" was not great either. Sounds like lat-lon CRS. So, I would not go back to that either.

Reply:
> I have couple of issues with this, none of them directly related to the main goals of the PR.
> 
> 1. `-c` for project/location creation gives `ERROR: Provide CRS to create a location` in 8.2. `-c XY` is what is needed for the described behavior. This might be there since 7.0.
> 2. I think XY is a CRS, no? It is just not georeferenced. Any opinions on this? Definitions?
> 3. The original "unprojected" was not great either. Sounds like lat-lon CRS. So, I would not go back to that either.

I agree that XY is a CRS, so why not `Creates a new XY GRASS project in the specified GISDBASE`?

Reply:
I believe that "No CRS" is more widely understood than "XY" (even in our bubble...).

Reply:
I'm not convinced the current text is 100% clear, but I also don't have any better suggestion. I agree that there is a perception that CRS must be georeferenced. Yet, many definitions even refer to the Earth specifically and that's obviously not true...

Reply:
> Well, QGIS has in GUI "No CRS". I am not sure XY is immediately understood outside of GRASS?

I still don't really understand it myself, and have medium to small geospatial knowledge. I stumbled upon it because of the testing infrastructure, so I just used it as it was required without understanding it.